perfmetrics/scripts/gsheet/gsheet.py

```diff
@@ -23,6 +24,11 @@ def write_to_google_sheet(worksheet: str, data) -> None:
   Raises:
     HttpError: For any Google Sheets API call related errors
   """
+
+  # Update the gsheet only if it runs as a part of daily Kokoro tests.
+  if os.getenv('KOKORO_JOB_TYPE') != "RELEASE" :
```

Comment:
We should update the gSheet for any valid JOB_TYPES supported by KOKORO. So that, in case we change, job_type of our kokoro job, we don't need to change here everytime.

Reply:
fixed